crates/turbopack-ecmascript/src/chunk/source_map.rs

```diff
@@ -131,7 +135,7 @@ impl AssetReference for EcmascriptChunkSourceMapAssetReference {
                     EcmascriptChunkEntrySourceMapAsset {
                         chunk_path: path,
                         code: item.code_vc,
-                        hash: item.hash,
+                        id: item.chunk_item.id(),
```

Comment:
If the `item.hash` is already computed, why can't we just use it instead?

Will that force a new `EcmascriptChunkEntrySourceMapAssetVc` every time the hash changes?

Reply:
See followup PR.